schemas/session-replay/incremental-snapshot-record-schema.json

```diff
@@ -0,0 +1,34 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "$id": "incremental-snapshot-record-schema.json",
+    "title": "IncrementalSnapshotRecord",
+    "type": "object",
+    "description": "Schema of a Record type which contains mutations of a screen.",
+    "allOf": [
+        {
+            "$ref": "_common-record-schema.json"
+        },
+        {
+            "required": [
+                "type",
+                "data"
```

Comment:
It's a bit inconsistent if we require `data` in incremental record, but not in some other (focus + full snapshot). Is this intentional?

Reply:
nope...I missed that...should be required everywhere and `focus` doesn't have any data.